doc/monitoring/api_reference.rst

```diff
@@ -187,6 +296,119 @@ histogram
 summary
 ~~~~~~~
 
+A summary metric is used to collect statistical data
+about the distribution of values within the application.
+
+Each summary provides several measurements:
+
+*   total count of measurements
+*   sum of measured values
+*   values at specific quantiles
+
+Similar to histograms, a summary also operates with value ranges. However, unlike histograms,
+it uses quantiles (defined by a number between 0 and 1) for this purpose. In this case,
+it is not required to define fixed boundaries. For summary type, the ranges depend
+on the measured values and the number of measurements.
+
+Suppose the example series of measurements are sorted in ascending order:
+1, 4, 6, 7, 7, 8, 8, 8.
+
+Quantiles:
+
+*   Quantile 0 is the value of the first, minimum element. In this example, it's 1.
+*   Quantile 1 is the value of the last, maximum element. In this example, it's 8.
+*   Quantile 0.5 is the value of the median element. In this example, it's 7. This means that the smaller
+    half of the measurements is a range of values from 1 to 7. The larger one is a range of values from 7 to 8.
+
+Note that calculating quantiles requires resources, so it makes sense to calculate no
+more than one, for example: 0.95 -- the majority of measurements.
+
+With a large number of measurements per second, a significant amount of memory is required to
+store them all. The array is compressed to reduce memory consumption. The degree of compression is determined by
+an acceptable error rate. In application, error rates mostly from 1% to 10%. This means that a
+quantile of 0.50 with a 10% error from the example above returns a value in the range of ``6.65...7.35`` instead of 7.
```

Comment:
It's a bit inconsistent that range is in double ticks and other values are not.

Reply:
Fixed